test/unit/math/prim/arr/meta/contains_vector_test.cpp

```diff
@@ -8,6 +8,6 @@ TEST(MetaTraits, contains_vector) {
 
   EXPECT_TRUE(contains_vector<std::vector<double> >::value);
   EXPECT_TRUE(contains_vector<std::vector<int> >::value);
-  EXPECT_TRUE(contains_vector<std::vector<const double> >::value);
```

Comment:
Items in a vector must be assignable

https://stackoverflow.com/questions/17313062/vector-of-const-objects-giving-compile-error